reportportal/values.yaml

```diff
@@ -400,6 +400,24 @@ ingress:
   #   - reportportal.k8.com
   #   secretName: reportportal.k8.com-tls
 
+  # If you wish to use istio ingress gateways
+  # set ingress.enable to false
+  # set ingress.istio.enable true
+  # configure hostnames with ingress.hosts[]
+  istio:
```

Comment:
Please move it outside ingress values and name it something like `istioIngress`. Since putting it inside 'Ingress' section makes these two options logically connected. I would expect chart automatically disable nginx in case of istio enabled, but this not the case.